src/ConfigTrait.php

```diff
@@ -44,8 +44,6 @@ trait ConfigTrait
      */
     public function readConfig($files = ['config.php'], $format = 'php')
     {
-        $this->config = [];
```

Comment:
If you remove this line, then you have to make sure $this->config is empty array at the beginning. So maybe let's set it as empty array in class properties - line 29.
or in line 83 do like this:
`$this->config = array_merge_recursive($this->config ?: [], $tempConfig);`